src/Notepad.java

```diff
@@ -1319,38 +1310,37 @@ public void compileFile() {
                                 System.getProperty(
                                         "user.home") + "\\err.txt"));
                 String s = "";
-                boolean b=false;
-                while ((s = din.readLine())!=null) {
+                boolean b = false;
+                while ((s = din.readLine()) != null) {
                     jTextArea2.append(s + "\n");
-                    b=true;
+                    b = true;
                 }
                 din.close();
-                if(!b){
+                if (!b) {
                     jTextArea2.setText("No Syntax Error");
                 }
             } catch (Exception e) {
             }
-        }
```

Comment:
This is a C+Java compiler

Reply:
Yes I Know. But Whenever You Save The Files. Your dir and name Variables Were Not Changing.

Reply:
And Thats Why It Was Giving Null Pointer Exception Whenever You Compile And Run Any File That Has Been Created And Saved With Notiler  Menu.